This fixes configure issues with recent autoconf, e.g:
  autoreconf: Entering directory `Modules/_ctypes/libffi'
  autoreconf: configure.ac: not using Gettext
  autoreconf: running: aclocal --force
  configure.ac:26: error: m4_copy: won't overwrite defined macro: _AC_ARG_VAR_PRECIOUS
  configure.ac:26: the top level

The problem is still present in python-2.6.5 but fixed in python-svn.

Upstream-Status: Accepted [python-svn]

Signed-off-by: Antonio Ospite <ospite@studenti.unina.it>
Acked-by: Martin Jansa <Martin.Jansa@gmail.com>
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
Signed-off-by: Saul Wold <sgw@linux.intel.com>


2011/09/29
Rebased for python 2.7.2
Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>


Index: Python-2.7.2/Modules/_ctypes/libffi/Makefile.am
===================================================================
--- Python-2.7.2.orig/Modules/_ctypes/libffi/Makefile.am
+++ Python-2.7.2/Modules/_ctypes/libffi/Makefile.am
@@ -2,7 +2,7 @@
 
 AUTOMAKE_OPTIONS = foreign subdir-objects
 
-SUBDIRS = include testsuite man
+SUBDIRS = include
 
 EXTRA_DIST = LICENSE ChangeLog.v1 ChangeLog.libgcj configure.host \
 	src/alpha/ffi.c src/alpha/osf.S src/alpha/ffitarget.h \
@@ -34,8 +34,6 @@ EXTRA_DIST = LICENSE ChangeLog.v1 Change
 	libtool-version ChangeLog.libffi m4/libtool.m4 \
 	m4/lt~obsolete.m4 m4/ltoptions.m4 m4/ltsugar.m4 m4/ltversion.m4
 
-info_TEXINFOS = doc/libffi.texi
-
 ## ################################################################
 
 ##
